packages/shared/src/contexts/OnboardingContext.tsx

```diff
@@ -85,7 +85,7 @@ export const OnboardingContextProvider = ({
     trackEvent({
       event_name: AnalyticsEvent.ClickArticleAnonymousCTA,
       target_id: ExperimentWinner.ArticleOnboarding,
-      extra: JSON.stringify({ origin }),
+      extra: JSON.stringify({ origin: window.origin }),
```

Comment:
Just a note, origin is actually never the "origin" as in http origin.
It's more an internal measure.

In this case it should be one of these two according to analytics:
```
article page, article modal
```

I think maybe by accident this got lost?

Reply:
In this case it was passed as `window.origin` for quite some time. So this is a bug?

Reply:
Because there is no `origin` prop or variable in `OnboardingContext`

Reply:
Yep 100% a bug, should not be the case.
I'll also check with Ido how it impacted analytics.

Reply:
Below are the correct values, but there's no significant impact anyway at this point.
Thanks for pointing that out. Let's fix that when possible (not urgent).
<img width="758" alt="image" src="https://github.com/dailydotdev/apps/assets/1993245/0dec0cf0-75cf-4475-b8e6-408404c68021">

Reply:
Thanks for checking ill add it as a seperate task then 👍

Reply:
@rebelchris we can merge this then?